tests/integration/actions/exec/test_stdout_vault.py

```diff
@@ -0,0 +1,64 @@
+"""Tests for exec, mode stdout, parameters set using the CLI.
+
+A variety of formats are tested here, each with a different
+placement of the vault command. Although none result in a vaulted
+file, the intent is to ensure the entire vault command is passed
+to ansible vault within the execution environment.
+"""
+
+import pytest
+
+from ..._interactions import SearchFor
+from ..._interactions import UiTestStep
+from ..._interactions import add_indices
+from .base import BaseClass
+
+
+class ShellCommand(UiTestStep):
+    """A shell command."""
+
+    search_within_response = SearchFor.PROMPT
+
+
+VAULT_COMMAND = (
+    "ansible-vault encrypt_string --vault-password-file a_password_file 'foobar'"
+    " --name 'the_secret'"
+)
```

Comment:
Why not use something like `echo` or similar/wrapper? I'd prefer an executable with low overhead for the test to be faster, plus running ansible internals is not essential for this test.

Reply:
As an example, you could do something like printing the args back and then just compare the output in tests here. You could even make those assertions more precise:
```suggestion
VAULT_COMMAND = (
    "python -c 'import sys; print(sys.argv)' -- ansible-vault encrypt_string "
    "--vault-password-file a_password_file 'foobar' --name 'the_secret'"
)
```

Reply:
The use of vault here was intentional, this will be _the_ documented approach for both galaxy and vault until such time navigator has entry points(if ever). I like the tests proving a suggested use case. Related: https://github.com/ansible/ansible-navigator/issues/620

Reply:
that said........... that second example is really cool.  I may add another test file, with just that so as issues come in, if they do, there is a sane place to put them.

Reply:
In that case, consider using [pytest-bdd](https://github.com/pytest-dev/pytest-bdd) (in the future) for documented behaviors.

Reply:
will pass on the second suggestion here, only because asserting against the output of vault proves the params were passed all the way through to it